bin/ember-template-lint.js

```diff
@@ -481,22 +250,11 @@ async function run() {
   let hasTodos = options.includeTodo && results.todoCount;
   let hasUpdatedTodos = options.updateTodo;
 
-  let Printer = require('../lib/formatters/default');
   let printer = new Printer({
     ...options,
     hasResultData: hasErrors || hasWarnings || hasTodos || hasUpdatedTodos,
   });
   printer.print(results, todoInfo);
 }
 
-// exports are for easier unit testing
-module.exports = {
-  _parseArgv: parseArgv,
-  _expandFileGlobs: expandFileGlobs,
-  _getFilesToLint: getFilesToLint,
-  _getPossibleOptionNames: getPossibleOptionNames,
-};
-
-if (require.main === module) {
```

Comment:
Moved the exported helper functions from this file to a new helper file `lib/helpers/cli.js` since it's tricky in ESM to have a binary script that can also be imported.